src/shrutiS/BankingSystem.java

```diff
@@ -0,0 +1,63 @@
+package shrutiS;
+
+public class BankingSystem {
+	int debitCount;
+	int creditCount;
+	int balanceCount;
+	//double totalDebit;
+	//double totalCredit;
+	static int totalDebitTimes;
+	static int totalCreditTimes;
+	static int totalBalanceTimes;
+
+	void debitAmount(double debitAmount) {
+		// System.out.println("Amount debited : "+ debitAmount);
+		//totalDebit = totalDebit + debitAmount;
```

Comment:
why you comments this?
we need this operations. think when you do debit from account then amt is deductible from our account right. then why you comments this part. same for other method.